src/redux/reducers/paginationReducer.js

```diff
@@ -0,0 +1,18 @@
+import { GET_CURRENT_PAGE } from 'redux/actions/types';
+
+const initialState = {
+  currentPage: 1,
+  episodesPerPage: 12
+};
+
+export default function getCurrentPage(state = initialState, action) {
+  switch(action.type) {
+    case GET_CURRENT_PAGE:
+      return {
+        currentPage: action.payload,
+        episodesPerPage: 12
```

Comment:
how the `episodesPerPage` can be change? on what type of action? Maybe until it's hardcoded value (not coming from API and it's not selected from API data) consider to pass it into component directly instead push it to the store

Reply:
Right, moved directly to the `<App />` component.